src/frontend/Felles/Personopplysninger/Adressehistorikk.tsx

```diff
@@ -27,6 +21,18 @@ const FetTekst = styled.span`
 
 const MAX_LENGDE_ADRESSER = 5;
 
+const TekstMedTagWrapper = styled.div`
+    display: flex;
+    gap: 0.5rem;
+    align-items: center;
+`;
+
+const BostedsadresserBeskrivelseWrapper = styled.div`
```

Comment:
```suggestion
const ColumnFlexBox = styled.div`
```
Eventuelt `KolonneFlexBox` `FlexBoxKolonne` `FlexBoxColumn`